contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -97,5 +97,8 @@ install() {
 				type mark_hostonly >/dev/null 2>&1 && mark_hostonly @systemdunitdir@/zfs-import-$_item.service
 			fi
 		done
+		inst_multiple -o \
+			systemd-ask-password systemd-tty-ask-password-agent \
+			${NULL}
```

Comment:
nit: for consistency with the rest of the script let's use `dracut_install` for these and list one per-line, the same as above.